modules/role/manifests/st2.pp

```diff
@@ -22,4 +20,12 @@
       include ::profile::hubot
     }
   }
+
+  if $_enterprise_token {
+    include ::profile::st2flow
+    include ::profile::st2rbac
+    include ::profile::enterprise_auth_backend_ldap
+  } else {
+    include ::profile::auth_backend_pam
```

Comment:
Please do do both PAM and LDAP for enterprise. 
In case of evaluation, folks may be interested in only Flow (+ some upcoming enterprise features) and not ready to setup ldap. Let them do it at their pace, and PAM will have some auth in place out-of-the box.